.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,14 +1,25 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Propose an idea for a feature or an enhancement
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**What problem does this address, if any?**
+<!--
+If this feature or enhancement is related to a current problem or pain point,
+please describe it. For example, "I'm always frustrated when ..." or "It is
+currently difficult to ...".
+-->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+**What is the solution?**
+<!--
+Please outline the feature or enhancement that you want and how it addresses any
+problem identified above.
+-->
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+**What are the alternative approaches?**
```

Comment:
I always personally find this question a bit annoying. Do you think it's needed? It could at least be marked 'optional'.

Reply:
I really like the idea of this being optional too. Like you, I've found it to be more of a nuisance.

Reply:
I think it's important to ask "Should this really be in Core?" in some form. We have a very mature plugin ecosystem and so should lean on that when possible. Perhaps it would be better to outright ask that question, or to ask about how the feature aligns with the philosophy (https://wordpress.org/about/philosophy/) which already covers things like "Design for the majority" and "Provide features that 80% of people will use".

Reply:
In theory the Core team should always ask these questions but in practice we're busy (and, frankly, too polite 😛) so I think it's helpful if the reporter does some upfront consideration of these things.

Reply:
That's a great point. Since it's already in other parts of the system, I say we try it. At the least, people can state if it's not as high impact potentially and it might help set some expectations.

Reply:
Could we use some checkboxes? Something like this might make it easy:

```
## How does this fit in with our philosophy (https://wordpress.org/about/philosophy/)
- [ ] Is the feature something that 80% or more of end users will appreciate and use?
- [ ] Does the solution fit the philosophy of 'Decisions, not Options'?
```

Reply:
I added a little preamble which points to https://wordpress.org/about/philosophy/ and has a faux checklist. It's more of a gentle nudge in the right direction than a strict check. I don't want the issue template to be too... bossy. 😀